src/physics/BeamSolver.cpp

```diff
@@ -969,8 +969,10 @@ TInputLine *TBeamSolver::ParseFile(int& N)
 					ShowError(M);
 				}
 
-				for (j=0;j<Lines[i].N;j++)
+				for (j=0;j<Lines[i].N;j++){
 					Lines[i].S[j]=ReadWord(L,j+2);
```

Comment:
@skutsaev I am unclear about these lines. Line 973 seems unnecessary. Line 974 I changed to remove the UpperCase(), because Linux has a case-sensitive file system so one can't upcase a file name and expect it to be found. It would be ideal to call UpperCase only when a comparison is needed. I don't know all the places where the comparison are happening. Are there other places besides IsKeyWord, ParseDistribution, Parse, ParseSpchType, ParseFile, ParsePIT, ParseOptions, etc.? That is, is there comparisons outside of BeamSolver.cpp? Thanks.

Reply:
@robnagler I don't think there are any comparisons outside BeamSolver.cpp, since it should be the only place where the input file is parsed. I can check all occurrences when I return in a couple of weeks.

Reply:
@skutsaev I updated all comparisons in BeamSolver.cpp. Our tests pass. I'll merge this now, because I think it works the way it is supposed to. LMK if you run into issues.